pyteomics/proforma.py

```diff
@@ -448,7 +456,10 @@ def resolve(self, name=None, id=None, **kwargs):
             defn = self.database['MOD:{:05d}'.format(id)]
         else:
             raise ValueError("Must provide one of `name` or `id`")
-        mass = float(defn.DiffMono)
+        try:
+            mass = float(defn.DiffMono)
```

Comment:
Thank you!
It appears that on Python 2 the new test fails because the expected exception is not raised. Where on Python 3 `defn.DiffMono` is `None` here, on Python 2 it is equal to `229.162932` for `"TMT6plex"`. I'm not sure of the exact reason yet, maybe you see what is happening?